### PR TITLE
fixed some minor errors

### PR DESCRIPTION
**Pull request by A-Basirat**
Fixed some minor errors

### CODE REVIEW

File: dashboard/src/components/DetailedView/DetailedView.js [hunk 1]
```diff
@@ -87,39 +89,48 @@ const DetailedView = ({ allPollutants = [] }) => {
...
-              onPollutantSelect={id => {
-                const currPollutant = pollutants.filter(pollutant => Number(pollutant.id) === Number(id));
-                setSelectedPollutant(currPollutant[0]);
-              }}
-            />
-
-            <div className="pollutant-details-container">
-              <PollutantDetails pollutant={selectedPollutant} />
-            </div>
+            <PollutantList onPollutantSelect={setSelectedPollutant} />
```
Comment (CoreyPatrickPDX) [inline dashboard/src/components/DetailedView/DetailedView.js:92]: Where are you getting the pollutants?

File: dashboard/src/components/DetailedView/DetailedView.js [hunk 2]
```diff
@@ -87,39 +89,48 @@ const DetailedView = ({ allPollutants = [] }) => {
...
+                  ))}
+                </select>
+              </div>
+
+              <button
+                className="search-btn detail-search-btn"
+                onClick={onSearchButtonClick}
+              >
+                Search
+              </button>
```
Comment (CoreyPatrickPDX) [inline dashboard/src/components/DetailedView/DetailedView.js:123]: Are the search input layouts the same between the DetailedView and MainView?

File: dashboard/src/components/MainView/MainView.js [hunk 3]
```diff
@@ -8,32 +8,39 @@ import PollutantDetails from "../Pollutants/PollutantDetails";
...
 
+    // const countryResponse = await getCountryData("Mexico");
+    // console.log(countryResponse);
+    // const cityResponse = await getCityData("London");
+    // console.log(cityResponse);
+    // const result = await getCityData(searchedCity)
+    //   .then(response => response.results);
+    // setCityData(result[0]); //getCityData will now return either an element of the results array (station w the highest # of recorded parameters)
+    // //or the response as a whole if the results array is empty or missing - this needs to be adjusted as a result.
+  };
```
Comment (CoreyPatrickPDX) [inline dashboard/src/components/MainView/MainView.js:43]: We should think about deleting out this commented out code just so we don't forget to later

**Review by CoreyPatrickPDX — COMMENTED** [review]
Overall it looks good! It looks like it has a few elements from the dataCleanup branch I pushed earlier

File: dashboard/src/components/DetailedView/DetailedView.js (hunk 1 above)
Comment (A-Basirat) [reply dashboard/src/components/DetailedView/DetailedView.js:92]: There is a pollutant list on the PollutantList.js file!

File: dashboard/src/components/DetailedView/DetailedView.js (hunk 2 above)
Comment (A-Basirat) [reply dashboard/src/components/DetailedView/DetailedView.js:123]: Yeah, but we don't need that for the historical view. All we gotta do for the historical view is search for historical data on all 12 pollutants in the list for a specific date range.

File: dashboard/src/components/MainView/MainView.js (hunk 3 above)
Comment (A-Basirat) [reply dashboard/src/components/MainView/MainView.js:43]: once we're done with the implementation, we gotta clean up all the code.

File: dashboard/src/components/DetailedView/DetailedView.js (hunk 1 above)
Comment (CoreyPatrickPDX) [reply dashboard/src/components/DetailedView/DetailedView.js:92]: Awesome!

File: dashboard/src/components/DetailedView/DetailedView.js (hunk 2 above)
Comment (CoreyPatrickPDX) [reply dashboard/src/components/DetailedView/DetailedView.js:123]: Right! We have the separate form for the historical view.

File: dashboard/src/components/MainView/MainView.js (hunk 3 above)
Comment (CoreyPatrickPDX) [reply dashboard/src/components/MainView/MainView.js:43]: I agree, I think mainly one person with eslint and prettier enabled to follow the airbnb style guide should go through and make sure everything is compliant.

**Pull request closed without merging**